src/codemagic/tools/app_store_connect.py

```diff
@@ -148,6 +154,71 @@ def _delete_resource(self, resource_manager, resource_id: ResourceId, ignore_not
             else:
                 raise AppStoreConnectError(str(api_error))
 
+    @cli.action('list-testflight-builds',
+                BuildArgument.APPLICATION_ID_OPTIONAL,
+                BuildArgument.EXPIRED,
+                BuildArgument.NOT_EXPIRED,
+                BuildArgument.BUILD_ID,
+                BuildArgument.PRE_RELEASE_VERSION,
+                BuildArgument.PROCESSING_STATE,
+                BuildArgument.BUILD_VERSION,
+                BuildArgument.ORDERING,
+                BuildArgument.REVERSE)
+    def list_testflight_builds(self,
+                              application_id: Optional[Union[str, ResourceId]] = None,
+                              expired: Optional[bool] = None,
+                              not_expired: Optional[bool] = None,
+                              build_id: Optional[Union[str, ResourceId]] = None,
+                              pre_release_version: Optional[str] = None,
+                              processing_state: Optional[BuildProcessingState] = None,
+                              build_version: Optional[str] = None,
+                              ordering: Optional[BuildOrdering] = None,
+                              reverse: Optional[bool] = False,
+                              should_print: bool = True) -> List[Build]:
+        """
+        List Testflight builds.
+        """
+        try:
+            expired_value = get_binary_arguments_value(expired, not_expired)
+        except:
+            raise BuildArgument.NOT_EXPIRED.raise_argument_error('Using mutually exclusive flags "--expired" and "--not-expired"')
+
+        builds_filter = self.api_client.builds.Filter(
+            app=application_id,
+            expired=expired_value,
+            id=build_id,
+            processing_state=processing_state,
+            version=build_version,
+            pre_release_version__dot__version=pre_release_version,
+        )
+        kwargs = {'ordering': ordering, 'reverse': reverse}
+        return self._list_resources(builds_filter, self.api_client.builds, should_print, **kwargs)
+
+    @cli.action('get-latest-testflight-build-number',
+        BuildArgument.APPLICATION_ID,
+        BuildArgument.PRE_RELEASE_VERSION)
+    def get_latest_testflight_build_number(self,
+                                           application_id: Union[str, ResourceId],
+                                           pre_release_version: Optional[str] = None,
+                                           should_print: bool = False) -> None:
```

Comment:
should print False so that only build number is echoed to use as
```
export BUILD_NUMBER=$(app_store_connect get-latest-testflight-build-number '1501865727')
```

return to keep python api